### ISSUE TITLE
ImportError: cannot import name 'StrEnum' from 'enum'

### ISSUE THREAD

**Issue by Katehuuh** [theredsix/cerebellum#11]
```cmd
Traceback (most recent call last):
  File "C:\browser-use\examples.py", line 2, in <module>
    from cerebellum import AnthropicPlanner, BrowserAgent, BrowserAgentOptions, pause_for_input
  File "C:\browser-use\venv\lib\site-packages\cerebellum\__init__.py", line 1, in <module>
    from .browser import *
  File "C:\browser-use\venv\lib\site-packages\cerebellum\browser.py", line 11, in <module>
    from enum import StrEnum
ImportError: cannot import name 'StrEnum' from 'enum' (C:\Python\Python310\lib\enum.py)
```
Python: `3.10.8`

Error supposedly related to an older incompatible version. 
https://github.com/theredsix/cerebellum/blob/ce83dbe23fc97d7c88a8b739706bca54eed844ae/python/pyproject.toml#L52
Could this please be made retrocompatible to 3.10.x?

---

Additional suggestion: Support OpenAI-compatible endpoints `OPENAI_API_KEY` + `OPENAI_API_BASE` (useful for local LLM).

**Comment by theredsix** [comment]
@scm-aiml thoughts on the python backport?

**Comment by scm-aiml** [comment]
@theredsix this should be an easy fix, and should be able to support python `3.10`. I will take a quick change and try just setting it like this https://github.com/theredsix/cerebellum/blob/ce83dbe23fc97d7c88a8b739706bca54eed844ae/python/src/cerebellum/browser.py#L90

**Comment by theredsix** [comment]
Hi @Katehuuh, this should be fixed in v0.2.3.

**Comment by Katehuuh** [comment]
When run the google example python again:
```cmd
(venv) C:\browser-use>python examples.py
Traceback (most recent call last):
  File "C:\browser-use\examples.py", line 2, in <module>
    from cerebellum import AnthropicPlanner, BrowserAgent, BrowserAgentOptions, pause_for_input
  File "C:\browser-use\venv\lib\site-packages\cerebellum\__init__.py", line 1, in <module>
    from .browser import *
  File "C:\browser-use\venv\lib\site-packages\cerebellum\browser.py", line 14, in <module>
    from cerebellum.utils import parse_xdotool, pause_for_input
  File "C:\browser-use\venv\lib\site-packages\cerebellum\utils.py", line 2, in <module>
    import termios
ModuleNotFoundError: No module named 'termios'

(venv) C:\browser-use>pip install termios
ERROR: Could not find a version that satisfies the requirement termios (from versions: none)
ERROR: No matching distribution found for termios
```

FYI `termios` will not work because Windows os is not a POSIX platform.
